config/ember-try.js

```diff
@@ -74,6 +75,8 @@ module.exports = async function () {
           },
         },
       },
+      embroiderSafe(),
+      embroiderOptimized(),
```

Comment:
Actually, we would like to allow these two jobs to fail for now, could you update it?